moped-editor/src/components/GridTable/Filters.js

```diff
@@ -637,14 +352,33 @@ const Filters = ({
         </Grid>
       </Grid>
 
-      {Object.keys(filterParameters).map((filterId) => {
-        const isValidInput = checkIsValidInput(filterParameters[filterId]);
+      {filterParameters.map((filter, filterIndex) => {
+        const { field: fieldName, operator, value } = filter;
+
+        /* Get field configuration and values need for inputs */
+        const fieldConfig = filtersConfig.fields.find(
+          (field) => field.name === fieldName
+        );
+        const { label, type } = fieldConfig ?? {};
+        const operators = fieldConfig?.operators;
```

Comment:
nah not nits to me - this is great stuff and thank you for taking such a detailed look at this! 🙏 I'll address these comments today.

Reply:
This took me a second, but, yes, I see it now. Yeah, I overlooked how the list of operators are keys in `FiltersCommonOperators`. I'm going to update so that the form input just accesses the label name using the operator key. I can get rid of a helper this way too. 😈